### DISCUSSION TITLE
求助，make clean all编译老是提示错误

### DISCUSSION THREAD

**Discussion by Comymx** [filecoin-project/community-china#114] [新手上路]
所有依赖环境都安装了，按照官方操作，执行make clean all 就提前这个错误，求大神指点一下
rm -rf  build/.filecoin-install build/.update-modules  lotus lotus-miner lotus-worker lotus-shed lotus-gateway lotus-seed lotus-pond lotus-townhall lotus-fountain lotus-chainwatch lotus-bench lotus-stats lotus-pcr lotus-health lotus-wallet lotus-keygen testground tvx lotus-sim
make -C extern/filecoin-ffi/ clean
make[1]: 进入目录“/home/kuang1/lotus/extern/filecoin-ffi”
make[1]: *** 没有规则可制作目标“clean”。 停止。
make[1]: 离开目录“/home/kuang1/lotus/extern/filecoin-ffi”
Makefile:312: recipe for target 'clean' failed
make: [clean] Error 2 (已忽略）
git submodule update --init --recursive
touch build/.update-modules
make -C extern/filecoin-ffi/ .install-filcrypto
make[1]: 进入目录“/home/kuang1/lotus/extern/filecoin-ffi”
make[1]: *** 没有规则可制作目标“.install-filcrypto”。 停止。
make[1]: 离开目录“/home/kuang1/lotus/extern/filecoin-ffi”
Makefile:37: recipe for target 'build/.filecoin-install' failed
make: *** [build/.filecoin-install] Error 2

**Comment by tang952745** [comment]
我有台机子也是这个错误，你解决了嘛，加个vx一起交流下

**Comment by luozhanxin** [comment]
`filecoin-ffi` 依赖项目拉取成功了吗？

**Comment by FCJack996** [comment]
直接删除extern文件，然后先git submodule update --init --recursive防止克隆不完整，再使用make clean，如果还不行可以再交流，我构建了四五次，有一些经验

**Reply by Joymerry** [reply]
我一直都是报fatal: clone of 'https://github.com/filecoin-project/fil-blst.git' into submodule path '/Users/joymerry/Desktop/filecoin/lotus/extern/test-vectors/gen/extern/fil-blst' failed
Failed to clone 'gen/extern/fil-blst' a second time, aborting
Failed to recurse into submodule path 'extern/test-vectors'
请问该怎么处理？

**Comment by Chenhongwe** [comment]
单卡单U单台封装效率7T，软硬一体方案，详询v15088695255
